app/models/image.rb

```diff
@@ -7,11 +7,14 @@ class Image < ActiveRecord::Base
   belongs_to :item, inverse_of: :images
   before_destroy :delete_image_from_cloudinary, unless: "Rails.env.test?"
 
+  def public_image_id
+    cloudinary_id.split('/').last.split('.').first rescue nil
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.